vhost-device-console/src/backend.rs

```diff
@@ -42,11 +42,14 @@ pub enum Error {
     ThreadPanic(String, Box<dyn Any + Send>),
     #[error("Error using multiple sockets with Nested backend")]
     WrongBackendSocket,
+    #[error("Invalid uds file")]
+    InvalidUdsFile,
 }
 
 #[derive(PartialEq, Eq, Debug)]
 pub struct VuConsoleConfig {
     pub socket_path: PathBuf,
+    pub vm_sock: String,
```

Comment:
`uds_path` is better IMHO, so it's clear that it can be used only with the new `uds` backend